package.json

```diff
@@ -92,7 +92,7 @@
     "minimatch": "^3.0.4",
     "open": "^6.3.0",
     "ora": "^3.4.0",
-    "portfinder": "^1.0.13",
+    "portfinder": "1.0.21",
```

Comment:
Why not `^1.0.21`?

Reply:
Because 1.0.22 introduced a breaking change https://github.com/http-party/node-portfinder/issues/85

Reply:
Sorry, wrong link. https://github.com/http-party/node-portfinder/issues/89 describes better what we're experiencing.

Reply:
portfinder has released a fix. I've updated the code to read `^1.0.23`. This PR is no longer urgent. https://www.npmjs.com/package/portfinder/v/1.0.23